uninstall.php

```diff
@@ -111,7 +118,11 @@
 	// Get all options.
 	$give_option_names = $wpdb->get_col(
 		$wpdb->prepare(
-			"SELECT option_name FROM {$wpdb->options} where option_name LIKE '%%%s%%'",
+			"
+			SELECT option_name
+			FROM {$wpdb->options}
+			WHERE option_name LIKE '%%%s%%'
```

Comment:
Are there too many `%`s in the WHERE clause? It looks like we are using wildcards along side a merge syntax, so the merged value would be `%%give%%`, which would be redundant wildcards - correct?